adapter/operations.go

```diff
@@ -16,13 +16,21 @@ package adapter
 
 import "context"
 
-const (
-	OperationDescriptionKey  = "description"
-	OperationVersionKey      = "version"
-	OperationTemplateNameKey = "templateName"
-	OperationServiceNameKey  = "serviceName"
+var (
+	NoneVersion = []Version{"none"}
 )
 
+type Version string
+
+type Operation struct {
+	Type        int32     `json:"type,string,omitempty"`
+	Description string    `json:"description,omitempty"`
+	Versions    []Version `json:"versions,omitempty"`
```

Comment:
what does it mean to have multiple versions in one operation?

Reply:
I assume that having multiple versions of an operation is intended to capture the use case of supporting multiple versions of something like a workload (e.g. bookinfo v1.7.1, bookinfo v1.17.2)

Reply:
Yes that is exactly the one

Reply:
In one and the same operation? Isn't this the operation initiated by one request to the API? Would that mean that in one request, multiple versions would be installed?

Reply:
@mgfeller The version is/must be a part of the API request. Yes it is going to be changed as well